docs/docs/axes/styling.md

```diff
@@ -55,3 +46,68 @@ The majorTick configuration is nested under the ticks configuration in the `majo
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
 | `enabled` | `boolean` | `false` | If true, major ticks are generated. A major tick will affect autoskipping and `major` will be defined on ticks in the scriptable options context.
+
+## Indexable Options
+
+Indexable options also accept an array in which each item corresponds to the element at the same index. Note that this method requires to provide as many items as data, so, in most cases, using a [function](#scriptable-options) is more appropriated if supported.
```

Comment:
Needs a slight update for the looping behaviour that just got added

Reply:
Updated slightly, does that make sense?